pkg/logs/tailers/windowsevent/tailer_test.go

```diff
@@ -267,6 +270,227 @@ func (s *ReadEventsSuite) TestBookmarkNewTailer() {
 	// if tailer started from bookmark correctly, there should only be s.numEvents
 }
 
+func TestInitialBookmarkSeeding(t *testing.T) {
+	// Test that verifies the fix for the amnesia bug:
+	// When a tailer starts with no bookmark, it creates an initial bookmark
+	// from the most recent event and saves it immediately, even if no events
+	// are processed before shutdown.
+
+	// Setup test environment
+	channelPath := "dd-test-channel-initial-bookmark"
+	eventSource := "dd-test-source-initial-bookmark"
+
+	ti := eventlog_test.GetAPITesterByName("Windows", t)
+	if ti == nil {
+		t.Skip("Windows API tester not available")
+	}
```

Comment:
add the new tests to `ReadEventsSuite` so they can take advantage of the common setup and teardown done by the suite